packages/flutter_tools/test/integration/daemon_mode_test.dart

```diff
@@ -38,7 +38,11 @@ void main() {
       const ProcessManager processManager = LocalProcessManager();
       process = await processManager.start(
           <String>[flutterBin, '--show-test-device', 'daemon'],
-          workingDirectory: tempDir.path);
+          workingDirectory: tempDir.path,
+          environment: <String, String>{
+            'FLUTTER_WEB': 'false',
```

Comment:
Aren't these lines and similar lines in other files cruft? `true' is still the only value that's ever meaningful in this iteration of the patch.